spyvm/model.py

```diff
@@ -1035,6 +1035,13 @@ def short_atput0(self, space, index0, w_value):
     def size(self):
         return self._size
 
+    def as_string(self):
```

Comment:
`W_BytesObject.as_string` does not have this annotation. When would we not want to trace into this method?